mls-rs-crypto-awslc/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mls-rs-crypto-awslc"
-version = "0.10.0"
+version = "0.10.1"
```

```suggestion
version = "0.11.0"
```
This crate depends on `mls-rs-core` and that is a part of its public API (e.g. for the traits it defines), so bumping `mls-rs-core` to `0.18.0` is a breaking change for this crate.